packages/client/lib/Chain.js

```diff
@@ -180,6 +180,14 @@ export default class Chain {
     return balance
   }
 
+  async getMempoolBalance (addresses) {
```

Comment:
These methods shouldn't be in the interface as they are not cross chain. Are you having issues accessing them internally in the providers using `getMethod`?

Reply:
@mattBlackDesign are you trying to get to `eth_pendingTransactions` or `parity_localTransactions` or `parity_pendingTransactions` via this method?

With above rpc calls, we can do something similar on eth side.